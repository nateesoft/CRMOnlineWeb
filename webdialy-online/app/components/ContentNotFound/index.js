import React from 'react';
import ImageNotfound from './notfound.png';

export default function ContentNotFound(props) {
  return (
    <div style={{ textAlign: 'center', color: '#ddd' }}>
      <img src={ImageNotfound} width={100} />
      <h1>{props.label}</h1>
    </div>
  );
}
