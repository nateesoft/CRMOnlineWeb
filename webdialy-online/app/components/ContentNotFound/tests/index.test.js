import React from 'react';
import renderer from 'react-test-renderer';

import ContentNotFound from '../index';

describe('ContentNotFound components', () => {
  it('test snapshot', () => {
    const component = renderer.create(<ContentNotFound />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
