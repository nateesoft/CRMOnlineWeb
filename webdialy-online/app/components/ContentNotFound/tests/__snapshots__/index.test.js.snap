// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentNotFound components test snapshot 1`] = `
<div
  className="makeStyles-root-1"
>
  <img
    alt=""
    src="IMAGE_MOCK"
    width={100}
  />
  <h1 />
</div>
`;
